crates/biome_js_analyze/src/lint/nursery/use_semantic_elements.rs

```diff
@@ -59,25 +63,83 @@ impl Rule for UseSemanticElements {
             let is_not_interative =
                 AriaRoles.is_not_interactive_element(element_name, extract_attributes);
             if is_not_interative {
-                return Some(AnyJsxAttribute::from(attr));
+                let static_value = attr.as_static_value().unwrap();
+                let role_value = static_value.as_string_constant().unwrap();
+                let candidates = AriaRoles.get_corresponding_element(role_value);
+
+                if let Some(elements) = candidates {
+                    let mut result_elements: Vec<String> = vec![];
+                    let mut result_attributes: Vec<(String, String)> = vec![];
+
+                    for element in elements {
+                        // Get only first attribute
+                        let attributes = element.1.first();
+
+                        result_elements.push(element.0.to_string());
+
+                        if let Some(attribute) = attributes {
+                            result_attributes
+                                .push((attribute.0.to_string(), attribute.1.to_string()));
+                        } else {
+                            result_attributes.push((String::new(), String::new()));
+                        }
+                    }
+
+                    return Some((
+                        AnyJsxAttribute::from(attr),
```

Comment:
Why do you cast this `attr` to `AnyJsxAttribute`? I don't think there's need for that.